### PR TITLE
create and implement user story7 (half way done)

### PR DESCRIPTION
**Pull request by clarahqy**

### CODE REVIEW

File: src/main/java/controller/ReviewController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package controller;
+
+import useCases.review_use_case.ReviewInputBoundary;
+import useCases.review_use_case.ReviewRequestModel;
+import useCases.review_use_case.ReviewResponseModel;
```
Comment (ryancys1234) [inline src/main/java/controller/ReviewController.java:5]: You don't need this import statement.

File: src/main/java/useCases/review_use_case/ReviewInteractor.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @Override
+    public boolean checkReviewCompleteness(ReviewRequestModel review) {
+        return review.isComplete();
+    }
+
+    @Override
+    public ReviewResponseModel addReview(ReviewRequestModel review){
+        User user = new RegularUser();
```
Comment (czz0120) [inline src/main/java/useCases/review_use_case/ReviewInteractor.java:37]: Need create User through UserFactory

File: src/main/java/useCases/review_use_case/ReviewInteractor.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        Review reviewObject = new Review(review.getRating(), review.getComment(), user);
+        //TODO: add this reviewObject to a User's review list
+        //TODO: implement ReviewGateway to save this reviewObject
+        String reviewString = "Review:\n" + "Comment: " + review.getComment() + "\n" +
+                "Rating: " + review.getRating() + "\n" + "User: " + review.getUser().toString();
+        LocalDateTime now = LocalDateTime.now();
+        return new ReviewResponseModel(reviewString, now.toString(), "added");
+
```
Comment (czz0120) [inline src/main/java/useCases/review_use_case/ReviewInteractor.java:45]: Does addReview need to report a successful view? seems like your interactor did not make use of the outputBoundary

**Comment by clarahqy** [comment]
This PR was closed because conflicts were too complex to resolve.

**Pull request closed without merging**